cdc/sink/mysql.go

```diff
@@ -915,6 +917,9 @@ func (s *mysqlSink) execDMLWithMaxRetries(
 					}
 				}
 				if err = tx.Commit(); err != nil {
+					if strings.Contains(err.Error(), "Error 9007: Write conflict") {
+						log.Info("write conflict happened", zap.Strings("query", dmls.sqls), zap.Any("args", dmls.values), zap.Int("bucket", bucket))
```

Comment:
We should pay attention to partition table id, but seems we can use row values to find data belongs to which partition.

Reply:
At the moment, `RowChangedEvent` does not record Partition-ID. maybe we should support this field

Reply:
For this case, the row belongs to which partition can be distinguished by row values.

Reply:
For this case, the row belongs to which partition can be distinguished by row values.